worlds/stardew_valley/items.py

```diff
@@ -602,6 +598,13 @@ def create_unique_filler_items(item_factory: StardewItemFactory, options: Starde
     return items
 
 
+def weapons_count(options: StardewValleyOptions):
```

Comment:
Very good method, I like the improvement of doing 5 + X to accomodate for future mods that may add even more. It's better than the one I suggested